wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -1521,7 +1649,13 @@ Start the dapp and navigate to it in your browser.
 yarn start
 ```
 
--->
+Your dapp should look similar to the following:
+
+<div class="row">
+  <div class="column">
+    <img src={require("../../../assets/starknet-tutorial-transfer-token.png").default} alt="Starknet transfer token" width="400" style={{border: "1px solid #DCDCDC"}} />
+  </div>
```

Comment:
Also, do you need to re-connect your wallet when you re-start the dapp for the token balance to appear? That should be noted, if so. What happens when you select "Transfer 1 STRK to yourself"?